src/test/java/com/uid2/admin/vertx/SharingServiceTest.java

```diff
@@ -574,6 +610,7 @@ void KeysetSetNew(Vertx vertx, VertxTestContext testContext) {
         };
     }*/
 
+    @Test
```

Comment:
@cody-constine-ttd I saw this test was missing the `@Test` annotation, so I've added it here